packages/lisk-transactions/src/utils/validation/validation.ts

```diff
@@ -136,3 +138,39 @@ export const validateFee = (data: string) =>
 
 export const isValidInteger = (num: unknown) =>
 	typeof num === 'number' ? Math.floor(num) === num : false;
+
+export const normalizeInput = (rawTransaction: TransactionJSON): void => {
```

Comment:
isn't this checked in validateTransaction?
also, it doesn't seem like this function is `normalizing` input

Reply:
in order to set `amount` and `fee` to `Bignum` upon initialization, we need to check if type is valid at this stage. for name how about `checkInput`?